example/lib/ide/ide.dart

```diff
@@ -25,18 +26,32 @@ class IDE extends StatefulWidget {
 }
 
 class _IDEState extends State<IDE> {
+  // ignore: prefer_final_fields
   bool _showLeftPane = true;
+  // ignore: prefer_final_fields
   bool _showRightPane = false;
+  bool _isAnalyzing = false;
 
   @override
   void initState() {
     super.initState();
     _initLspClient();
   }
 
+  @override
+  void dispose() {
+    widget.workspace.lspClient
+        .removeNotificationListener(_lspNotificationListener);
```

Comment:
Please be sure to set the line width to 120 characters. I noticed that the formatter changed some of the existing format.